Ghana/GHA_transform_load_dlt.ipynb

```diff
@@ -0,0 +1,160 @@
+    "            .withColumn('func_sub',\n",
+    "                when(col('Func2').startswith('70330'), 'judiciary')\n",
+    "                .when((col('FUnc1').startswith('703')) & (~col('Func2').startswith('70330')), 'public safety')\n",
+    "                .when(col('Func2').startswith('70435') | col('Func2').startswith('70432'), 'energy')\n",
+    "                # No breakdown of health spending into primary, secondary, tertiary etc\n",
+    "                # No breakdown of education spending into primary, secondary, tertiary etc\n",
+    "            )\n",
+    "            .withColumn('func',\n",
+    "                when(col('FUnc1').startswith('702'), 'Defence')\n",
+    "                .when(col('FUnc1').startswith('703'), 'Public order and safety')\n",
+    "                .when(col('FUnc1').startswith('704'), 'Economic affairs')\n",
+    "                .when(col('FUnc1').startswith('705'), 'Environmental protection')\n",
+    "                .when(col('FUnc1').startswith('706'), 'Housing and community amenities')\n",
+    "                .when(col('FUnc1').startswith('707'), 'Health')\n",
+    "                .when(col('FUnc1').startswith('709'), 'Education')\n",
+    "                .when(col('FUnc1').startswith('708'), 'Recreation, culture and religion')\n",
+    "                .when(col('FUnc1').startswith('710'), 'Social protection')\n",
+    "                .otherwise('General public services')\n",
+    "            )\n",
+    "            .withColumn('econ_sub',\n",
+    "                when(col('ECON1').startswith('25'), 'subsidies to production')                        \n",
+    "            )\n",
+    "            .withColumn('econ',\n",
+    "                when(col('ECON1').startswith('21'), 'Wage bill')\n",
+    "                .when(col('ECON1').startswith('23') | col('ECON1').startswith('31'), 'Capital expenditures')\n",
+    "                .when(col('ECON1').startswith('22'), 'Goods and services')\n",
+    "                .when(col('ECON1').startswith('25'), 'Subsidies')\n",
+    "                # missing information on Social benefits\n",
+    "                .when(col('ECON1').startswith('26'), 'Other grants and transfers')\n",
+    "                .when(col('ECON1').startswith('24'), 'Interest on debt')\n",
+    "                .otherwise('Other expenses')\n",
+    "            )\n",
+    "        )\n",
+    "    \n",
+    "@dlt.table(name=f'gha_boost_gold')\n",
+    "def boost_gold():\n",
+    "    return (dlt.read(f'gha_boost_silver')\n",
+    "            .withColumn('country_name', lit('Ghana')) \n",
+    "            .select('country_name',\n",
+    "                    'year',\n",
+    "                    'approved',\n",
+    "                    col('EXECUTED').alias('executed'),\n",
+    "                    'admin0',\n",
+    "                    'admin1',\n",
+    "                    'admin2',\n",
+    "                    col('geo_tmp').alias('geo1'),\n",
```

Comment:
why name it geo_tmp then only to alias later? Consider naming it geo1 in silver to avoid the aliasing here.